go.mod

```diff
@@ -42,6 +42,7 @@ require (
 require (
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/rs/zerolog v1.34.0
+	github.com/sergi/go-diff v1.3.1
```

Comment:
How are you using this one? 🤔

Reply:
Left over. I tried to use diffing but that didn't really work.